examples/todo-list/src/repositories/todo-list.repository.ts

```diff
@@ -49,4 +58,27 @@ export class TodoListRepository extends DefaultCrudRepository<
   public findByTitle(title: string) {
     return this.findOne({where: {title}});
   }
+
+  async find(
+    filter?: Filter<TodoList>,
+    options?: Options,
+  ): Promise<TodoListWithLinks[]> {
+    // Prevent juggler for applying "include" filter
+    // Juggler is not aware of LB4 relations
+    const include = filter && filter.include;
+    filter = filter && Object.assign(filter, {include: undefined});
+
+    const result = await super.find(filter, options);
+
+    // poor-mans inclusion resolver, this should be handled by DefaultCrudRepo
+    // and use `inq` operator to fetch related todos in fewer DB queries
+    if (include && include.length && include[0].relation === 'todos') {
```

Comment:
I remember we introduced a inclusion handler in the previous PoC(sorry still looking for the PR), shall we delegate these to the inclusion handler like [this PoC](https://github.com/strongloop/loopback-next/pull/2124/files#diff-e6b0a38c283bb45f8def7c9b39542851) ?

Reply:
I would like us to work incrementally.

In this spike, I focused on how to represent and describe navigational properties. I created this temporary resolver to allow me to test my proposal end-to-end.

I am expecting that we will replace this code with a real resolver once it's implemented. This is out of scope of my spike though.

AFAICT, there is no follow-up story to implement the resolver proposed in https://github.com/strongloop/loopback-next/pull/2124, do we need to create one? I can do that as part of this spike if you like.

Reply:
> In this spike, I focused on how to represent and describe navigational properties. I created this temporary resolver to allow me to test my proposal end-to-end.

Fair enough 👍 

> AFAICT, there is no follow-up story to implement the resolver proposed in #2124

That would be great thanks! No hurry we can create the implementation story after closing this spike.